### PR TITLE
Move examples to xcommon-cmake

### PR DESCRIPTION
**Pull request by xross**
Updates to examples:
- Moved build using xcommon-cmake
- (Removed build for xcore-200)
- Removed superfluous licence files
- Update Jenkinsfile for new build (run on linux agent (as macOS))
- Removed AN00136 as now maintained in a separate an00136 repo

(updated .gitignore)

Note, thhe documentation has not yet been updated, but looking at it it needs a major overhaul. Will be done in a different PR.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -11,39 +13,47 @@ pipeline {
...
-          // xcoreAllAppsBuild('examples')
-          xcoreAllAppNotesBuild('examples')
-          dir("${REPO}") {
-            runXdoc('doc')
+          checkout scm
+
+          dir("examples") {
+            withTools(params.TOOLS_VERSION) {
+              sh 'cmake -G "Unix Makefiles" -B build'
+              sh 'xmake -C build -j 8'
```
Comment (ed-xmos) [inline Jenkinsfile:43]: Small thing but seems most of the linux agents have 16 processors / 32 threads.

File: Jenkinsfile [hunk 2]
```diff
@@ -54,9 +64,9 @@ pipeline {
           }
         }
       }
-       post 
+       post
        {
-        failure 
+        failure
         {
           archiveArtifacts artifacts: "${REPO}/tests/logs/*.txt", fingerprint: true, allowEmptyArchive: true
```
Comment (ed-xmos) [inline Jenkinsfile:71]: This is neat - archive on fail only

**Review by ed-xmos — APPROVED** [review]
LGTM

**Review by danielpieczko — APPROVED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (xross) [reply Jenkinsfile]: This was running on macOS (not sure why). I'll try changing to Linux agent and remove [jobs] so it can use what it likes.

Comment (xross) [reply Jenkinsfile]: Moving to linux doesn't do much for the build step (it's only ~30 seconds) , but I can push the test step (--numprocesses) to reduce from 30 to 10 mins.

**Pull request merged**